src/views/forms/AccountManagementForms/DepositForm.tsx

```diff
@@ -130,6 +131,27 @@ export const DepositForm = ({ onDeposit, onError }: DepositFormProps) => {
     };
   }, []);
 
+  // Temporary default to Axelar USDC tokens while CCTP is down.
+  // Revert this change once CCTP is back up.
+  useEffect(() => {
+    const USDCTokenAddresses = [
```

Comment:
[nit] if we use a set we get constant time lookups instead of having to use `includes`

Reply:
good call!